Formula/node.rb

```diff
@@ -56,30 +56,11 @@ def install
 
     if build.with? "npm"
       resource("npm").stage buildpath/"npm_install"
-
-      # make sure npm can find node
-      ENV.prepend_path "PATH", bin
```

Comment:
Reading your other review comment I guess this issue is more likely caused by the new npm installtion logic in this formula and not by the necessarry changes to `Language::node` to make `std_npm_install_args`compatible with npm@5.
Sorry If I was not clear enough in my first post, but all my comments were based on proposed changes to `Language::node`. I haven't had time to look in depth into the changes of npm installation logic in this PR. Maybe the discussion about the necessary changes to  `Language::node` should be separated over to an issue/pr in Homebrew/brew.